xfields/fieldmaps/interpolated_src/atomicadd.clh

```diff
@@ -0,0 +1,22 @@
+#ifndef _ATOMICADD_CLH_
```

Comment:
Instead of duplicating this file you can just get it from xtrack as done here:
https://github.com/xsuite/xfields/blob/d3c4130646f2d2c2e06562d43a705a36e815ae13/xfields/fieldmaps/interpolated.py#L161

Reply:
`xt.general._pkg_root.joinpath('headers/atomicadd.h')` was already used in beambeam3d.py. I just removed these two lines from beambeam3d.h:
"""
/include_file ../../fieldmaps/interpolated_src/atomicadd.clh for_context opencl
/include_file ../../fieldmaps/interpolated_src/atomicadd.h for_context cpu_serial cpu_openmp
"""